base/main_test_bucket_pool.go

```diff
@@ -513,6 +513,10 @@ func (tbp *TestBucketPool) createTestBuckets(numBuckets, bucketQuotaMB int, buck
 	// create required number of buckets (skipping any already existing ones)
 	for i := 0; i < numBuckets; i++ {
 		testBucketName := fmt.Sprintf(tbpBucketNameFormat, tbpBucketNamePrefix, i, bucketNameTimestamp)
+		// shorten the name for rosmar buckets since they are in memory only
+		if UnitTestUrlIsWalrus() {
+			testBucketName = fmt.Sprintf("bucket%d", i)
+		}
```

Comment:
I'm OK with this change, but not really the profiling one.

It's probably best just to split this unrelated stuff even if it adds a few mins of effort because it makes merging or reviewing easier.